modules/apm/src/main/java/org/elasticsearch/telemetry/apm/internal/metrics/AbstractInstrument.java

```diff
@@ -58,7 +58,7 @@ String getDescription() {
         return description;
     }
 
-    void setProvider(@Nullable Meter meter) {
+    public void setProvider(@Nullable Meter meter) {
```

Comment:
why does this need to be public? That causes the otel API to leak.

Reply:
Changed back to package private.